spec/integration/launcher_spec.rb

```diff
@@ -133,7 +133,7 @@
       end
     end
 
-    it 'user_data_dir option should restore cookies', sinatra: true do
+    it_fails_firefox 'user_data_dir option should restore cookies', sinatra: true do
```

Comment:
This spec works in local development environment, however fails on CI.

Official puppeteer doesn't check this spec. https://github.com/puppeteer/puppeteer/blob/523aa0aafa9b2bf2c14e0ba416e4100f73bafa0a/test/launcher.spec.ts#L275
We also skip it.